extensions/agroal/deployment/src/main/java/io/quarkus/agroal/deployment/AgroalProcessor.java

```diff
@@ -311,4 +325,200 @@ HealthBuildItem addHealthCheck(AgroalBuildTimeConfig agroalBuildTimeConfig) {
         return new HealthBuildItem("io.quarkus.agroal.runtime.health.DataSourceHealthCheck",
                 agroalBuildTimeConfig.healthEnabled, "datasource");
     }
+
+    @BuildStep
+    void registerMetrics(AgroalBuildTimeConfig agroalBuildTimeConfig,
+            BuildProducer<MetricBuildItem> metrics) {
+        Metadata activeCountMetadata = Metadata.builder()
+                .withName("agroal.active.count")
+                .withDescription("Number active of connections. This connections are in use and not available to be acquired.")
```

Comment:
```suggestion
                .withDescription("Number of active connections. These connections are in use and not available to be acquired.")
```

Reply:
Good catch, I admit I was mostly just copying javadoc from Hibernate codebase without thinking too hard about it. There are some typos apparently.